integration/tests/cook/util.py

```diff
@@ -243,23 +243,19 @@ def all_instances_killed(response):
 
 
 def wait_for_job(cook_url, job_id, status, max_delay=120000):
-    """
-    Wait for the given job's status to change to the specified value.
-    Returns an up-to-date job description object on success,
-    and raises an exception if the max_delay wait time is exceeded.
-    """
-    job_id = unpack_uuid(job_id)
+    """Wait for the given job's status to change to the specified value."""
+    return wait_for_jobs(cook_url, [job_id], status, max_delay)[0]
 
+def wait_for_jobs(cook_url, job_ids, status, max_delay=120000):
```

Comment:
Missing docstring

Reply:
Do we require docstrings on everything in here? How valuable would one be? This method is pretty self-descriptive.